src/data/faq_de.json

```
@@ -293,12 +293,15 @@
                         ]
                     },
                     {
-                        "title": "Kann ein vollständiger Impfstatus in der App als Berechtigung zum Reisen dienen?",
+                        "title": "Kann mein Zertifikat in der App als Berechtigung zum Reisen dienen?",
                         "anchor": "vac_cert_privileges_travel",
                         "active": true,
                         "textblock": [
-                            "Ob Sie mit einem vollständigen Impfschutz beispielsweise reisen dürfen, hängt von den Bestimmungen des jeweiligen Landes ab.",
-                            "Sie können unter anderem diese <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>Webseite</a> benutzen, um Ihre Reise innerhalb der EU zu planen."
+                            "Das digitale Impfzertifikat der EU kann nicht nur in Deutschland genutzt werden. Das in Deutschland ausgestellte digitale Impfzertifikat der EU kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt auch für Genesenen-Zertifikate und Testzertifikate. Die Gültigkeit Ihres Zertifikats kann ab Corona-Warn-App-Version 2.6 auch innerhalb der App vor Reiseantritt geprüft werden.",

```

```suggestion
                            "Das digitale Impfzertifikat der EU kann nicht nur in Deutschland genutzt werden. Das in Deutschland ausgestellte digitale Impfzertifikat der EU kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt auch für Genesenenzertifikate und Testzertifikate. Die Gültigkeit Ihres Zertifikats kann ab Corona-Warn-App-Version 2.6 auch innerhalb der App vor Reiseantritt geprüft werden.",
```
Align to app spelling of "Genesenenzertifikate"